chains/evm/contracts/onRamp/CommitOnRamp.sol

```diff
@@ -45,25 +43,22 @@ contract CommitOnRamp is Ownable2StepMsgSender, BaseOnRamp {
   /// It performs critical validation to ensure message integrity and proper sequencing.
   /// @param message Decoded MessageV1 payload for verification.
   /// @param 2nd parameter is messageId, unused.
-  /// @param feeToken Fee token used for the message.
-  /// @param feeTokenAmount Amount of fee token provided.
-  /// @param verifierArgs Opaque verifier-specific args.
+  /// @param 3rd parameter is feeToken, unused.
+  /// @param 4th parameter is feeTokenAmount, unused.
+  /// @param 5th parameter, is verifierArgs (opaque verifier-specific args), unused.
   /// @return verifierReturnData Verifier-specific encoded data
   function forwardToVerifier(
     MessageV1Codec.MessageV1 calldata message,
     bytes32, // messageId, unused
-    address feeToken,
-    uint256 feeTokenAmount,
-    bytes calldata verifierArgs
+    address, // feeToken, unused
+    uint256, // feeTokenAmount, unused
+    bytes calldata // verifierArgs, unused
   ) external returns (bytes memory verifierReturnData) {
     // For EVM, sender is expected to be 20 bytes.
     address senderAddress = address(bytes20(message.sender));
```

Comment:
do we need an explicit check for length here?

Reply:
I think it is safe to leave as is, already handled by CCVProxy & was like this before.